config/samples/kustomization.yaml

```diff
@@ -1,4 +1,7 @@
 ## Append samples you want in your CSV to this file as resources ##
 resources:
-  - v1alpha1_perses.yaml
+  - perses.dev_v1alpha1_perses.yaml
+  - perses.dev_v1alpha1_persesdashboard.yaml
+  - perses.dev_v1alpha1_persesdashboard-2.yaml
```

Comment:
`perses.dev_v1alpha1_persesdashboard-2.yaml` shouldn't be removed ?

Reply:
Well it is a file that exists in config/samples. The only file listed here before `v1alpha1_perses.yaml`, which does no longer exist (was removed in commit #45e3c38800a4200f00f74badbf8aebf28867aa82).

I can remove the reference to `perses.dev_v1alpha1_persesdashboard-2.yaml` if you want. Should the file itself also be removed? It has been added in #45e3c38800a4200f00f74badbf8aebf28867aa82 as well, by @jgbernalp  (It seems it is mostly a copy of `perses.dev_v1alpha1_persesdashboard.yaml`?)

Reply:
looks weird to me yeah.
Sorry if you find my question dumb, I'm not that familiar with the operator. I would wait what @jgbernalp will say about it

Reply:
No worries. I don't have any opinion on these samples either. I merely saw that the kustomization.yaml was outdated and referenced things that no longer exist, so I updated it with everything that exists in the directory 🤷

Reply:
We can remove it is the same as the first example so does not add any value at the moment.

Reply:
Removed both the file and the reference.